src/org/jitsi/impl/neomedia/rtp/FrameDesc.java

```diff
@@ -183,6 +195,19 @@ public int getStart()
     }
 
     /**
+     * Returns whether or not the first sequence number of this frame
+     * is known (conclusively)
+     *
+     * @return true if we know the first sequence number of this frame, false
+     * otherwise
+     */
+    public boolean firstSequenceNumberKnown()
```

Comment:
We can use this here as well: https://github.com/jitsi/jitsi-videobridge/blob/f3425d22f5b594891a5a47fc50cb40da25ce9efb/src/main/java/org/jitsi/videobridge/cc/SimulcastController.java#L738